vortex-ipc/src/messages.rs

```diff
@@ -17,13 +22,17 @@ pub(crate) enum IPCMessage<'a> {
 }
 
 pub(crate) struct IPCContext<'a>(pub &'a ViewContext);
+
 pub(crate) struct IPCSchema<'a>(pub &'a DType);
+
 pub(crate) struct IPCChunk<'a>(pub &'a ViewContext, pub &'a ArrayData);
+
 pub(crate) struct IPCArray<'a>(pub &'a ViewContext, pub &'a ArrayData);
 
 impl FlatBufferRoot for IPCMessage<'_> {}
+
 impl WriteFlatBuffer for IPCMessage<'_> {
-    type Target<'a> = fb::Message<'a>;
+    type Target<'a> = fbi::Message<'a>;
```

Comment:
I think `fb` is ok if it's the default `fb` for the crate. In this case, IPC. Other crates' fb should be namespaced